### PR TITLE
feat: ✨ Burmese translation

### PR DESCRIPTION
**Pull request by enson0131**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 💡 Background and solution
Burmese locales were missing.

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |  Added locales for Burmese translation.  |
| 🇨🇳 Chinese | 添加缅甸语    |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### CODE REVIEW

File: components/calendar/locale/mm_MM.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import mmMM from '../../date-picker/locale/mm_MM';
```
Comment (afc163) [inline components/calendar/locale/mm_MM.ts:1]: https://github.com/react-component/pagination/pull/518/files

my_MM.ts or mm_MM.ts ?

File: components/calendar/locale/mm_MM.ts
Comment (afc163) [inline components/calendar/locale/mm_MM.ts:1]: my_MM.ts or mm_MM.ts ?

**Comment by yoyo837** [comment]
react-component/pagination#517

File: components/calendar/locale/mm_MM.ts
Comment (enson0131) [reply components/calendar/locale/mm_MM.ts:1]: Ok, let me correct it

**Comment by enson0131** [comment]
> [react-component/pagination#517](https://github.com/react-component/pagination/issues/517)

When will it be updated in npm?

**Comment by yoyo837** [comment]
> > [react-component/pagination#517](https://github.com/react-component/pagination/issues/517)
> 
> When will it be updated in npm?

Waiting for the owner to release it.

**Comment by enson0131** [comment]
> Waiting for the owner to release it.

When will it be updated? We will use this feature as soon as possible.

**Comment by yoyo837** [comment]
While waiting for the upstream release, the current PR should be sent to the feature branch, which will release a minor version at the end of the month. https://ant.design/changelog#release-schedule

**Comment by yoyo837** [comment]
```
+ rc-picker@3.3.4
+ rc-pagination@3.3.1
```

`feature` branch Please.

**Comment by enson0131** [comment]
> ```
> + rc-picker@3.3.4
> + rc-pagination@3.3.1
> ```
> 
> `feature` branch Please.

merge into feature branch ? not master branch?

**Comment by yoyo837** [comment]
The guide is here https://ant.design/docs/react/i18n-cn#%E5%A2%9E%E5%8A%A0%E8%AF%AD%E8%A8%80%E5%8C%85.

**Comment by enson0131** [comment]
> The guide is here https://ant.design/docs/react/i18n-cn#%E5%A2%9E%E5%8A%A0%E8%AF%AD%E8%A8%80%E5%8C%85.

okok, Thanks

**Comment by enson0131** [comment]
> The guide is here https://ant.design/docs/react/i18n-cn#%E5%A2%9E%E5%8A%A0%E8%AF%AD%E8%A8%80%E5%8C%85.

should I create new PR ？

**Comment by yoyo837** [comment]
> > The guide is here https://ant.design/docs/react/i18n-cn#%E5%A2%9E%E5%8A%A0%E8%AF%AD%E8%A8%80%E5%8C%85.
> 
> should I create new PR ？

It's a nice try.

**Comment by enson0131** [comment]
> Burmese locales were missing.



> > > The guide is here https://ant.design/docs/react/i18n-cn#%E5%A2%9E%E5%8A%A0%E8%AF%AD%E8%A8%80%E5%8C%85.
> > 
> > 
> > should I create new PR ？
> 
> It's a nice try.

ok, I create new PR, https://github.com/ant-design/ant-design/pull/41366

**Comment by yoyo837** [comment]
Move to #41276

**Pull request closed without merging**